bitbots_body_behavior/src/bitbots_body_behavior/decisions/ball_close.py

```diff
@@ -1,10 +1,12 @@
+import math
 from dynamic_stack_decider.abstract_decision_element import AbstractDecisionElement
 
 
 class BallClose(AbstractDecisionElement):
     def __init__(self, blackboard, dsd, parameters=None):
         super(BallClose, self).__init__(blackboard, dsd, parameters)
         self.ball_close_distance = parameters.get("distance", self.blackboard.config['ball_close_distance'])
+        self.ball_close_angle = parameters.get("angle", math.pi)
```

Comment:
Maybe use `self.blackboard.config['ball_reapproach_ange']` as default.

Reply:
No, because the decision is used elsewhere just with the distance and the default case should not include any angle constraints.